Assets/Scripts/enemyBullet.cs

```diff
@@ -0,0 +1,26 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class enemyBullet : MonoBehaviour
+{
+    public float dieTime;
+    public float damage;
+    public Transform player;
+    public float bulletSpeed;
+    void Start()
+    {
+        Destroyer();
+    }
+
+    private void OnCollisionEnter2D(Collision2D collision)
+    {
+        Destroy(gameObject);
+    }
+
+    void Destroyer()
```

Comment:
Tak się zastanawiam, czy musiałeś to rozdzielić na oddzielną funkcje. Jak to jest jedna linijka plus nie będziesz jej w taki sposób nigdzie powtarzał. To czy nie czytelniej by było po prostu wrzucić tego co jest w tej funkcji do 'void Start()'?

Reply:
chciałem dać jedną metodę dla 2 momentów usunięcia obiektu, ale nie miało sensu czekać 3 sekund po uderzeniu w inny obiekt więc zmieniłem, ale metoda dla pozostałej możliwości została ;)

Reply:
Też w sumie uważam że to jest jedną linijka kodu i no też jestem za tym by dać to po prostu do start